nbformat/validator.py

```diff
@@ -333,6 +528,14 @@ def iter_validate(
     relevant notebook format schema.
 
     Returns a generator of all ValidationErrors if not valid.
+
+
+    Notes
+    -----
+    To fix: For security reason this should not ever mutate it's arguments, and
+    should not ever try to validate a mutated or modified version of it's notebook.
```

Comment:
```suggestion
    should *never* try to validate a mutated or modified version of its notebook.
```
("its notebook" or "the notebook" ? I'm not sure what `its` references in this context)